sklearn/linear_model/_base.py

```diff
@@ -46,6 +46,30 @@
 # intercept oscillation.
 
 
+def _deprecate_normalize(normalize, default):
+    if normalize == 'deprecated':
+        _normalize = default
+    else:
+        _normalize = normalize
+
+    if default or (normalize != 'deprecated' and normalize):
+        warnings.warn(
+            "'normalize' was deprecated in version 0.24 and will be"
+            " removed in 0.26. If you still wish to normalize use Pipeline "
+            " with a StandardScaler in a"
+            " preprocessing stage:"
+            "  model = make_pipeline(StandardScaler(),"
+            " {type(self).__name__}())", FutureWarning
+        )
+    elif (normalize != 'deprecated' and not normalize):
+        warnings.warn(
+            "'normalize' was deprecated in version 0.24 and will be"
+            " removed in 0.26.", FutureWarning
```

Comment:
We should add something like "to suppress this warning, don't set the parameter and leave it to its default value."